src/api/include/migraphx/migraphx.h

```diff
@@ -123,17 +123,21 @@ typedef const struct migraphx_program* const_migraphx_program_t;
 typedef struct migraphx_operation* migraphx_operation_t;
 typedef const struct migraphx_operation* const_migraphx_operation_t;
 
+#ifdef MIGRAPHX_ENABLE_ONNX
```

Comment:
These need to be defined in a header though. Perhaps a config.h header that sets values from cmake variables.

Reply:
Nevermind, this wont be needed if we use python variables instead.